test/unit/Assistant.spec.ts

```diff
@@ -20,12 +20,14 @@ import {
   createDummyAssistantThreadStartedEventMiddlewareArgs,
   createDummyAssistantUserMessageEventMiddlewareArgs,
   createDummyMessageEventMiddlewareArgs,
+  proxyquire,
   wrapMiddleware,
 } from './helpers';
 import { team } from './helpers/events';
 
 async function importAssistant(overrides: Override = {}): Promise<typeof import('../../src/Assistant')> {
-  return rewiremock.module(() => import('../../src/Assistant'), overrides);
+  const absolutePath = path.resolve(__dirname, '../../src/Assistant');
+  return proxyquire(absolutePath, overrides);
 }
```

Comment:
🔍 question: Is an absolute path needed for indirect imports?

🗣️ thought: IMO this makes the testing pattern more clear - `overrides` override whatever's exported from `absolutePath` - was most curious!